lib/galaxy/authnz/managers.py

```diff
@@ -126,7 +127,8 @@ def _parse_custos_config(self, config_xml):
             'client_id': config_xml.find('client_id').text,
             'client_secret': config_xml.find('client_secret').text,
             'redirect_uri': config_xml.find('redirect_uri').text,
-            'realm': config_xml.find('realm').text}
+            'realm': config_xml.find('realm').text,
+            'enable_idp_logout': config_xml.findtext('enable_idp_logout', 'false').lower() == "true"}
```

Comment:
The `galaxy.util.asbool` function is a potential replacement for this: https://github.com/galaxyproject/galaxy/blob/0204c9b65e1dac407046112b7db7901a36eed971/lib/galaxy/util/__init__.py#L925

Reply:
Good suggestion, will do.